jdaviz/configs/cubeviz/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -53,6 +53,12 @@ class SpectralExtraction(PluginTemplateMixin, DatasetSelectMixin,
     extracted_spec_available = Bool(False).tag(sync=True)
     overwrite_warn = Bool(False).tag(sync=True)
 
+    aperture_method_items = List(['exact', 'subpixel', 'center']).tag(sync=True)
+    aperture_method_selected = Unicode('exact').tag(sync=True)
```

Comment:
are all three of these functional now (I don't see it used anywhere).  Assuming we will have three options, then changing #2676 from a switch to a dropdown will definitely make sense.

Reply:
@bmorris3 and I focused on using exact so I have not tested the other methods. I can check that for the next commit.